identity/aziot-identityd/src/error.rs

```diff
@@ -19,84 +19,87 @@ impl Error {
-        match self {
-            Error::Authentication => f.write_str("authentication error"),
-            Error::Authorization => f.write_str("authorization error"),
-            Error::DeviceNotFound => f.write_str("device identity not found"),
-            Error::DPSClient(_) => f.write_str("DPS client error"),
-            Error::HubClient(_) => f.write_str("Hub client error"),
-            Error::ModuleNotFound => f.write_str("module identity not found"),
-            Error::Internal(_) => f.write_str("internal error"),
-            Error::InvalidParameter(name, _) => write!(f, "parameter {:?} has an invalid value", name),
-        }
-    }
+	fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+		match self {
+			Error::Authentication => f.write_str("authentication error"),
+			Error::Authorization => f.write_str("authorization error"),
+			Error::DeviceNotFound => f.write_str("device identity not found"),
+			Error::DPSClient(_) => f.write_str("DPS client error"),
+			Error::HubClient(_) => f.write_str("Hub client error"),
+			Error::ModuleNotFound => f.write_str("module identity not found"),
+			Error::Internal(_) => f.write_str("internal error"),
+			Error::InvalidParameter(name, _) => write!(f, "parameter {:?} has an invalid value", name),
+		}
+	}
 }
 
 impl std::error::Error for Error {
-    fn source(&self) -> Option<&(dyn std::error::Error + 'static)> {
-        match self {
-            Error::Authentication |
-            Error::Authorization |
-            Error::DeviceNotFound |
-            Error::ModuleNotFound => None,
-            Error::DPSClient(err) |
-            Error::HubClient(err) => Some(err),
-            Error::Internal(err) => Some(err),
-            Error::InvalidParameter(_, err) => Some(&**err),
-        }
-    }
+	fn source(&self) -> Option<&(dyn std::error::Error + 'static)> {
+		match self {
+			Error::Authentication |
+			Error::Authorization |
+			Error::DeviceNotFound |
+			Error::ModuleNotFound => None,
+			Error::DPSClient(err) |
+			Error::HubClient(err) => Some(err),
+			Error::Internal(err) => Some(err),
+			Error::InvalidParameter(_, err) => Some(&**err),
+		}
+	}
 }
 
 #[allow(clippy::module_name_repetitions)]
 #[derive(Debug)]
 pub enum InternalError {
-    CreateCertificate(Box<dyn std::error::Error + Send + Sync>),
-    CreateHomeDir(std::io::Error),
-    InvalidUri(http::uri::InvalidUri),
-    LoadKeyOpenslEngine(openssl2::Error),
-    LoadDeviceInfo(std::io::Error),
-    LoadSettings(std::io::Error),
-    MasterIdentityKey(std::io::Error),
-    ParseDeviceInfo(toml::de::Error),
-    ParseSettings(toml::de::Error),
-    SaveDeviceInfo(std::io::Error),
-    SaveSettings(std::io::Error),
+	CreateCertificate(Box<dyn std::error::Error + Send + Sync>),
+	CreateHomeDir(std::io::Error),
+	InvalidUri(http::uri::InvalidUri),
+	LoadKeyOpenslEngine(openssl2::Error),
+	LoadDeviceInfo(std::io::Error),
+	LoadSettings(std::io::Error),
+	BadSettings(std::io::Error),
```

Comment:
Addition of BadSettings is the only non-spacing change in this file.

Reply:
Ok. Are we using rustfmt when submitting PR?